src/eth_test.rs

```diff
@@ -0,0 +1,13 @@
+use jsonrpc_core::{BoxFuture, Result};
+use jsonrpc_derive::rpc;
+use zksync_basic_types::H256;
+use zksync_types::transaction_request::CallRequest;
+
+///
+/// ETH namespace extension for the test node.
+///
+#[rpc]
+pub trait EthTestNodeNamespaceT {
+    #[rpc(name = "eth_sendTransaction")]
```

Comment:
Now that this is implemented, can you enable the following e2e test? `xdescribe` -> `describe`
```
describe("hardhat_impersonateAccount & hardhat_stopImpersonatingAccount" ...
```

Reply:
Where I can find this test?

Reply:
At the bottom of the following file `e2e-tests/test/hardhat-api.tests.ts`

You can also search for the `describe(...)` I posted above in the repo and it'll show up

Reply:
I see it, looks like my branch is outdated